packages/gatsby-plugin-emotion/README.md

```diff
@@ -12,7 +12,7 @@ guide](https://emotion.sh/docs/migrating-to-emotion-10#incremental-migration) fo
 ## Install
 
 ```shell
-npm install gatsby-plugin-emotion @emotion/core @emotion/styled
+npm install gatsby-plugin-emotion @emotion/core @emotion/styled @emotion/react
```

Comment:
`@emotion/core` was replaced by `@emotion/react`.

Reply:
yes, but in testing locally I got errors when not using core so I think core might still be required even if not using it directly

Reply:
I don't think so it's proper solution, because @emotion/core was renamed to `@emotion/react`, and `@emotion/core` is marked as removed - please look at `@emotion/core` v11.0.0, it haven't any functionality inside.

Reply:
ahh makes sense. pushed update